core/src/modules/configLoaders/app.ts

```diff
@@ -52,6 +59,9 @@ export async function loadApp(
 }
 
 export async function deleteApps(ids: string[]) {
+  // Since this method is only used when config is loaded and because we assume
+  // the db is ephemeral, we can target locked objects, even though this will
+  // always return zero objects when in config mode.
```

Comment:
Same here. This is duplicated in each of the relevant loaders.

Reply:
I think the comment here is a bit confusing:  This method is likely to be run on a "real" database - `grouparoo start,run,apply` all do it.  Apps that are no longer in the config will be deleted.